modules/arm_plugin/src/arm_converter/arm_converter_conv.cpp

```diff
@@ -59,14 +254,22 @@ template<> Converter::Conversion::Ptr Converter::Convert(const opset::ArmGroupCo
         ngraphWeightsShape[3],
         ngraphWeightsShape[4]
     }));
+
+    auto wInfoIt = node.get_rt_info().find("WeightsPrescaleInfo");
```

Comment:
BTW: If you need to store input/output specific runtime info you can do it in input/output as they also has `rt_info`

Reply:
Even though for ACL quantization info relates to tensors from the ngraph point of view these properties are internal to operation, so I prefer to retain them as part of the node itself.